Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -1375,11 +1403,32 @@ def run_polling_command(client, args: dict, cmd: str, upload_function: Callable,
             timeout_in_seconds=6000)
 
         command_result = CommandResults(scheduled_command=scheduled_command)
+    elif post_function:
+        # Validate the polling results
+        validate_submit_file_polling_results(command_result)
```

Comment:
here the code doesn't use the output of `validate`, is this intended?

Reply:
Yeah, because the whole point of the post function, in this case, is to validate the content of the report analysis, and if it finds any error messages, then it will raise an error, if no error is found, then we continue normally.